### DISCUSSION TITLE
FAQ entry: "other end is not receiving my MMS"

### DISCUSSION THREAD

**Discussion by foolishgrunt** [FossifyOrg/General-Discussion#494] [General]
I would have filed this as a bug, but the existence of the aformentioned FAQ entry suggests to me that it's a known issue that perhaps I need to understand better.

In a single MMS group that I'm a member of, nobody receives the messages I send from Fossify Messages (nor SMT Messages). Messages from the bundled LineageOS 'Messaging' app work fine, as well as from QKSMS. I do not know what is different about this particular group, nor do I know how to "reduce the message size" (as the FAQ suggests) of a plain text message.

Can anybody point me in the right direction?

**Comment by Aga-C** [comment]
Reducing message size is about the option in settings `Resize sent MMS images`.

If changing that option doesn't help, can you tell more about this problem? What are you sending in these MMS messages - images, videos or just text? Does it happen when sending MMS to other people/groups?

**Reply by foolishgrunt** [reply]
Sorry, somehow I missed your reply.

So far the only messages I have sent to this particular group are plain text, usually just a few words long - no images or anything. Since reducing the image size doesn't seem like it would have any effect in this case, I haven't tried.

And so far, no, I have not noticed messages failing to send to any other individuals or groups. However, I also do not receive any errors when sending messages to the problem group - it appears from my end that the message sent properly, but nobody else receives it. When I first became aware that this particular group wasn't receiving my messages, I made a new MMS group to test with my wife (Google Messages, same carrier) and brother (iMessage, different carrier) and they both confirmed that they received my message.

I don't really know what else to check. Are there any debug logs I could share?

**Reply by naveensingh** [reply]
@foolishgrunt Have you observed any "Error code: ..." like in https://github.com/FossifyOrg/Messages/issues/85 or https://github.com/FossifyOrg/Messages/issues/45?

**Reply by foolishgrunt** [reply]
No, I receive no error messages at all. It appears from my end that the message was sent properly, but nobody else in the chat receives it.

Anyway, per @Aga-C's response, it seems like the FAQ entry does not apply to my situation, so I am happy to file a new bug report.

**Reply by Aga-C** [reply]
@foolishgrunt Please create a new bug report providing as many details about this case as possible.
